examples/java/pom.xml

```diff
@@ -114,6 +114,37 @@
         </dependency>
       </dependencies>
     </profile>
+
+    <!--
+      This profile disable streaming integration tests which
+      have @Category(StreamingITs.class) annotation.
+
+      This profile can be abled on the command line
+      by specifying -P disable-streaming-ITs.
+    -->
+    <profile>
+      <id>disable-streaming-ITs</id>
```

Comment:
Once we have streaming ITs not in the examples directory we may want to move this up a directory or two.